build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:${project.gradleVersion}"
+        classpath 'com.android.tools.build:gradle:3.0.1'
```

Comment:
This should probably not be hardcoded. :)

Reply:
that's strange. I did not touch that file. Ill reedit and rebuild my project again!

thanx @misaochan

Reply:
I believe Android Studio prompts you and asks if you would like to "update Gradle version", upon which it modifies the file. I generally set that to "Not ask again". ;)